src/RazorSdk/Targets/Sdk.Razor.CurrentVersion.targets

```diff
@@ -201,6 +201,10 @@ Copyright (c) .NET Foundation. All rights reserved.
       $(GetCopyToOutputDirectoryItemsDependsOn)
     </GetCopyToOutputDirectoryItemsDependsOn>
 
+    <_RazorGenerateComponentDesignTimeDependsOn>
```

Comment:
Can you add a comment here for posterity?

Reply:
```suggestion
   <!-- Tooling calls `RazorGenerateComponentDesignTime` to produce `RazorComponentWithTargetPath` items that are required to correctly show colorization and completion. 
   The DependsOn targets for `RazorGenerateComponentDesignTime` is specified using this property.
    -->
    <_RazorGenerateComponentDesignTimeDependsOn>
```

Reply:
How about this